sdcm/db_stats.py

```diff
@@ -357,7 +357,12 @@ def _init_stats(self):
     def get_scylla_versions(self):
         versions = {}
         try:
-            versions_output = self.db_cluster.nodes[0].remoter.run('rpm -qa | grep scylla').stdout.splitlines()
+            node = self.db_cluster.nodes[0]
+            if node.is_rhel_like():
+                version_cmd = 'rpm -qa |grep scylla'
+            else:
+                version_cmd = "dpkg -l |grep scylla|awk '{print $2 \"-\" $3}'"
+            versions_output = node.remoter.run(version_cmd).stdout.splitlines()
             for line in versions_output:
```

Comment:
There is a property [`BaseNode.scylla_packages_installed`](https://github.com/scylladb/scylla-cluster-tests/blob/68fdc207f14722cec8826ff183d36a29a9d95584/sdcm/cluster.py#L2611) class introduced in #1778: 

You can iterate over it:

```python
    for line in self.db_cluster.nodes[0].scylla_packages_installed:
        ...
```

Or, if you really need to backport this change to 3.2 and 3.3, you can use the same commands as in the property implementation instead of grepping/awking output.